src/library/scala/collection/immutable/Seq.scala

```diff
@@ -74,3 +74,5 @@ object LinearSeq extends SeqFactory.Delegate[LinearSeq](List)
 trait LinearSeqOps[+A, +CC[X] <: LinearSeq[X], +C <: LinearSeq[A] with LinearSeqOps[A, CC, C]]
   extends Any with SeqOps[A, CC, C]
     with collection.LinearSeqOps[A, CC, C]
+
+abstract class AbstractSeq[+A] extends scala.collection.AbstractSeq[A] with Seq[A]
```

Comment:
Do we also want `AbstractLinearSeq` or `AbstractIndexedSeq`? What would be the gain of having them?

Reply:
Again, there are few (none?) new methods in these traits, so they should not cause additional forwarders to be emitted.